src/components/page-news-list/index.js

```diff
@@ -1,7 +1,45 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { NewsItemList } from "../";
 import { api } from '../../utils';
 
-export const PageNewsList = () => (
-    <NewsItemList ids={api.getItemsIds()}/>
-)
\ No newline at end of file
+const isArraysEqual = (arr1=[], arr2=[]) => arr1.toString() === arr2.toString();
+
+export class PageNewsList extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            data: undefined,
+        }
+
+        this.fetchItems = () => {
+            api.getItemsIds()
+                .then(data => { this.setState({ data }) })
+                .catch(err => { console.error(err) });
+        }
+    }
+
+    componentDidMount() {
+        this.fetchItems()
+    }
+
+    shouldComponentUpdate(nextProps, nextState) {
+        return !isArraysEqual(this.state.data, nextState.data)
+    }
+
+    render() {
+        console.log('RENDER');
```

Comment:
you prob shouldn't have it in a code. it won't hurt to remove this console.log before you merge